bindings/c/test/mako/async.cpp

```diff
@@ -169,6 +176,9 @@ void ResumableStateForRunWorkload::updateStepStats() {
 			stats.addLatency(OP_COMMIT, step_latency);
 		}
 		tx.reset();
+		if (args.transaction_timeout_tx > 0) {
```

Comment:
`Argument::setTimeoutIfEnabled(Transaction tx)`

Reply:
Done with the latest commit.